locales/de.json

```diff
@@ -0,0 +1,96 @@
+{
+	"File": "Datei",
+	"Quit": "Schliessen",
+	"Edit": "Bearbeiten",
+	"Undo": "Lösen",
+	"Redo": "Noch einmal",
+	"Cut": "Schneiden",
+	"Copy": "Kopieren",
+	"Paste": "Einfügen",
+	"Delete": "Löschen",
+	"Select all": "Alles markieren //?",
+	"View": "Wiedergabe",
+	"Reload": "Neu laden",
+	"Force Reload": "Neu laden // erzwingen?",
+	"Toggle Developer Tools": "Entwicklerwerkzeuge umschalten",
+	"Actual size": "Wahre Größe",
+	"Zoom in": "Hineinzoomen",
```

Comment:
`Vergrößern` would be the German, `Hineinzoomen` is Denglish.
@PrestonN In which context is this used?

Reply:
This is a menu entry. Under view.